### PR TITLE
Ming Cui_HW01 Submission

### PR DESCRIPTION
**Pull request by ming-cui**
I am familiar with R and (R)markdown. However, GitHub is quite foreign to me. It took me a while to set up. But the time investment is worthwhile. By following the lectures, I got to know how useful and versatile GitHub is. I can even build my personal website on GitHub. One thing to point out. The markdown syntax in GitHub and R can be rendered differently. For example, ##Header1 will be rendered successfully as a header in R. While in GitHub, it cannot be rendered as a formatted header unless I changed it to ## Header1. Moreover, by reading other materials about "fork", I got a better understanding of Git workflow. In short, I quite enjoy this course.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ and more...
 
 As a learner in social sciences, it is really the time to study computational methods.
 
-##Things I have learned so far
+## Things I have learned so far
 How to use Git workflow. I also found `<fork>` an interesting term. You may find [this website](https://www.atlassian.com/git/tutorials/comparing-workflows) useful for understanding `<fork>`.
```
Comment (rheuer83) [inline README.md:20]: I really like the use of `` here to refer to fork as a function!

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # hw01
 
-##About Me
+## About Me
```
Comment (rheuer83) [inline README.md:3]: The subheadings look really neat and visually pleasing!

File: README.md [hunk 3]
```diff
@@ -16,10 +16,10 @@ and more...
 
 As a learner in social sciences, it is really the time to study computational methods.
 
-##Things I have learned so far
+## Things I have learned so far
 How to use Git workflow. I also found `<fork>` an interesting term. You may find [this website](https://www.atlassian.com/git/tutorials/comparing-workflows) useful for understanding `<fork>`.
 
-##What is next?
+## What is next?
 Week 1 is pretty informative. I cannot wait for the incoming lectures.
```
Comment (rheuer83) [inline README.md:23]: This website looks really useful! Like you, I am familiar with R but not so much with github. To be honest, everything still feels quite unintuitive to me, so the link is very much appreciated!

File: README.md [hunk 4]
```diff
@@ -16,10 +16,10 @@ and more...
...
 
 ![](https://images.unsplash.com/photo-1555323912-3526a765162b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=600&q=60)
```
Comment (rheuer83) [inline README.md:25]: This website looks really useful! Like you, I am familiar with R but not so much with github. To be honest, everything still feels quite unintuitive to me, so the link is very much appreciated!

File: README.md (hunk 1 above)
Comment (rheuer83) [inline README.md:20]: This website looks really useful! Like you, I am familiar with R but not so much with github. To be honest, everything still feels quite unintuitive to me, so the link is very much appreciated!

**Review by rheuer83 — APPROVED** [review]
You seem to have implemented all the suggested elements and also succesfully submitted your assignment using github. Your biography could maybe use a little more detail, but I guess that's also not the main point of the exercise. Good job!

File: README.md [hunk 5]
```diff
@@ -1 +1,25 @@
...
+
+As a learner in social sciences, it is really the time to study computational methods.
+
+## Things I have learned so far
+How to use Git workflow. I also found `<fork>` an interesting term. You may find [this website](https://www.atlassian.com/git/tutorials/comparing-workflows) useful for understanding `<fork>`.
+
+## What is next?
+Week 1 is pretty informative. I cannot wait for the incoming lectures.
+
+![](https://images.unsplash.com/photo-1555323912-3526a765162b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=600&q=60)
```
Comment (aapalkova) [inline README.md:25]: Great job including a photo, maybe next time you can include some alternative text for the photo in the brackets as well!

File: README.md [hunk 6]
```diff
@@ -1 +1,25 @@
...
+My name is Ming Cui. I received my master's degree in business administration from University of Science and Technology of China. I am working on my master's in social sciences at UChicago.
+
+## Why this course?
+**Computational tools** are trending. You may find
+
+1. Computational Neuroscience
+1. Computational Biology
+1. Computational Sociology
+1. Computational Psychology
+1. Computational Economics
```
Comment (aapalkova) [inline README.md:13]: Your list is very clear and easy to read, maybe you can experiment with adding sub items next time.

File: README.md (hunk 1 above)
Comment (aapalkova) [reply README.md:20]: I second the use of ``, I always wondered how people made the text look that way and now I know!

File: README.md (hunk 5 above)
Comment (deblnia) [reply README.md:25]: +1 alt text is great for accessibility on the web

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |   X        |              |            |
| **Presentation**            |        X   |              |            |
| **Achievement, creativity** |    X       |              |            |
| **Reproducibility**         |   X        |              |            |

Remarks:

* Great work overall! You seem to have a good grasp on markdown syntax, and the git submission workflow, both of which are great. 😄 
* As a peer review noted, alt text is a really great thing to include for accessibility! (Just something to note going forward 😁 )

**Pull request closed without merging**